model/src/pyrenew/latent/infectionswithfeedback.py

```diff
@@ -129,7 +139,7 @@ def sample(
             Reproduction number.
         I0 : ArrayLike
             Initial infections, as an array
-            at least as long as the
+            of the same length as the generation
```

Comment:
It _should_ be  "at least as long", no, given that we subset `I0` appropriately below?

Reply:
addressed in https://github.com/CDCgov/multisignal-epi-inference/pull/124/commits/cf01560b46e8603bb161e995074b0fa64615cf0d